src/twisted/trial/_asynctest.py

```diff
@@ -131,43 +131,42 @@ def onTimeout(d):
     def __call__(self, *args, **kwargs):
         return self.run(*args, **kwargs)
 
-    def deferSetUp(self, ignored, result):
-        d = self._run("setUp", result)
-        d.addCallbacks(
-            self.deferTestMethod,
-            self._ebDeferSetUp,
-            callbackArgs=(result,),
-            errbackArgs=(result,),
-        )
-        return d
+    async def deferSetUp(self, result):
```

Comment:
As we change the signature and the return value of this method, I think is best to make it private.

```suggestion
    async def _deferSetUp(self, result):
```

Reply:
Fixed now.